File: Gamepython copy/organisms/plants/Belladonna.py
from organisms.Plant import Plant

class Belladonna(Plant):
    def __init__(self, world, position):
        super().__init__(world, position, strength=99, initiative=0, color="purple")

    def collision(self, other):
        other.die()
        self.die()
        self.world.log(f"{other.__class__.__name__} and Belladonna both died")

File: src/organisms/plants/Belladonna.java
import java.awt.Color;

import javax.swing.ImageIcon;

public class Belladonna extends Plant {
    public Belladonna(World world, Position pos) {
        super(world, pos);
        this.strength = 99;
        this.color = Color.PINK;
        this.icon = new ImageIcon("res/belladonna.png");
    }

    @Override
    public void collision(Organism attacker) {
        world.removeOrganism(attacker);
        world.log(attacker.getName() + " ate Belladonna and died!");
    }

    @Override
    public String getName() { return "Belladonna"; }
}